about.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header >
+        <nav >
+            <ul class="tallbar">
+                <li><a href=""><img src="img/Rectangle.png" alt=""></a></li>
+                <li><a href=""><img src="img/twitter.png" alt=""></a></li>
+                <li><a href="" class="English">English</a></li>
+                <li><a href="" class="My-Page">My Page</a></li>
+                <li><a href="" class="Logout">Logout</a></li>
+            </ul>
+        </nav>
+
+        <div class="hearder">
+            <div class="logo-container"><img src="img/mobile_menu 1.png" alt="" class="logo" onclick="showMenu()"></div>
+            <img src="img/logo (1).png" alt="" class="desk-logo">
+            <ul class="logo-menu">
+                <li ><a href="" class="About-color">About</a></li>
+                <li><a href=""></a>Program</li>
+                <li><a href=""></a>Join</li>
+                <li><a href=""></a>Sponsor</li>
+                <li><a href=""></a>News</li>
+                <li class="cc-campaign"><a href="" class="Campaign">CC_Campaign</a></li>
+                
+            </ul>
+            
+        </div>
+    </header>
+    <main>
+        <section class="about-page1">
+            <div class="about-first-page">
+                <div class="about-first-container">
+                    <div class="first-page-title-about">
+                        <h4 class="welcome-title">"Hello! Teaching Staff" </h4>
+                        <h1 class="main-title">COMPUTER SCIENCE PEDAGOGIG SEMINER 2023</h1>
```

Comment:
- [ ] Well, currently both titles are left-aligned, but as per the design, they should be center-aligned on all screen sizes. Please follow the design specifications and update your code to achieve the desired center alignment for these titles. This adjustment will contribute to a more visually appealing and consistent layout.